libraries/botframework-streaming/tests/PayloadSender.test.js

```diff
@@ -135,13 +135,11 @@ describe('PayloadTransport', function () {
             expect(ps.isConnected).to.equal(false);
         });
 
-        it('gracefully fails when trying to write before connecting.', function (done) {
+        it('gracefully fails when trying to write before connecting.', function () {
+            // When not connected, PayloadSender should not throw.
+            // It should drop the packet silently.
             const ps = new PayloadSender();
-            ps.disconnected = () => done();
             expect(ps.isConnected).to.equal(false);
-            ps.connect(new FauxSock());

```

Comment:
Removed bunches of line.

The test is for "**write before connecting**". It should not call `ps.connect()`.